x/auth/vesting/types/msgs.go

```diff
@@ -132,6 +133,10 @@ func (msg MsgCreatePeriodicVestingAccount) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid recipient address: %s", err)
 	}
 
+	if msg.StartTime < 1 {
+		return fmt.Errorf("invalid start time of %d, length must be greater than 0", msg.StartTime)
```

Comment:
Eliminate this check. Negative timestamps are perfectly fine. (UNIX Lore: DMR explicitly said that negative time_t is valid since he wanted to be able to represent his pre-1970 birthdate.)

Reply:
after further thought, having the vesting period start before the current blocktime is probably safe otherwise the vesting would malfunction during export -> import upgrades.